scripts/build.ps1

```diff
@@ -257,6 +257,16 @@ function Publish-Package
         Write-Verbose "Move-Item $coreCLR20PackageDir\$file $coreCLRExtensionsDir -Force"
         Move-Item $coreCLR20PackageDir\$file $coreCLRExtensionsDir -Force
     }
+	
+	# Note Note: If there are some dependencies for the TestHostProvider assemblies, those need to be moved too. 
+    $runtimeproviders = @("Microsoft.TestPlatform.TestHostProvider.dll", "Microsoft.TestPlatform.TestHostProvider.pdb")
```

Comment:
Do we want to keep the name generic? Should the name say that these are NET runtime providers?